src/index.ts

```diff
@@ -231,7 +235,7 @@ export { Options as OptionsFactory } from "./lib/factories/options"
 export { withRetries } from "./lib/withRetries"
 
 export const ApiClient = {
-  configure: (configuration: ApiClientConfig) =>
+  configure: (configuration: ApiClientConfiguration) =>
```

Comment:
in case we expose this type, please note the breaking change in the release notes

Reply:
We don't. This is probably also changing with the token refresh I'd double check when @lkappeler merges.